packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -254,7 +254,7 @@ export default mixins(
 			return !this.readOnly && this.isTriggerNode && (isWebhookBasedNode || isPollingNode || override);
 		},
 		workflow(): Workflow {
-			return this.getCurrentWorkflow();
+			return this.getCurrentWorkflow(true);
```

Comment:
https://www.notion.so/n8n/P1-Reviews-Expression-editor-e049831cb7b34f3d8c54bc6cddb522e8#b8a823877a474eb2952bbcb2db11d905

Reply:
this is an expensive operation to create new workflow.. and was huge performance benefit to get rid of it everytime we open NDV or make an operation.. is there a better fix here possible?

Reply:
Thanks. Moved the call to `saveAsNewWorkflow`.